### PR TITLE
Add Public Lecture Page

### PR DESCRIPTION
**Pull request by CannonLock**
Added the Page and the Navigation in md

### CODE REVIEW

**Comment by CannonLock** [comment]
Screen Shot 2021-07-22 at 12 27 40 PM

File: docs/public-lectures.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+The OSG User School is our premier training program for researchers,
```
Comment (osg-cat) [inline docs/public-lectures.md:3]: Let’s get a title on this page. “OSGVS21 Public Lectures” would be good.

File: docs/public-lectures.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+facilitators, and others who are interested in learning more about how to
+use High Throughput Computing to advance research.  This year's program is
```
Comment (osg-cat) [inline docs/public-lectures.md]: I like typographic quotes, etc., so be sure to check for those (e.g., “year’s”).

File: docs/public-lectures.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+online again and the application process is complete.  BUT we have some
+exciting news about sharing the OSG School with the entire community!
+
+**For the first time, the OSG Virtual School 2021 will open its core lectures
+live to the public**
```
Comment (osg-cat) [inline docs/public-lectures.md]: Missing period.

File: docs/public-lectures.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+* The basics of using HTCondor
+* Getting started on OSG, a national distributed HTC infrastructure for
+   Open Science
+* How to prepare software for use in HTC systems
+* Handling data (input and output) in HTC systems
+
+The public lectures are held daily, August 2-6, from 2:30-4:00 pm US
+Central Time.  
+
+The detailed schedule is at: [Lecture Schedule](https://indico.fnal.gov/event/49686/timetable/)
```
Comment (osg-cat) [inline docs/public-lectures.md]: Let’s try to make this more of a sentence … [The detailed schedule] is available on the Indico site. (Or something like that).

File: docs/public-lectures.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+In addition, the lectures will be recorded and posted on the School website
+eventually, along with the complete set of exercises.  While full
+participants also have priority access to experts, personalized learning
+plans, and small-group learning sessions, by opening lectures up to the
+public, a great deal of the School content will be available to everyone.
+
+REGISTRATION
```
Comment (osg-cat) [inline docs/public-lectures.md]: Make this a 2nd or 3rd level heading.

File: docs/public-lectures.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+Registration is free but required: [Register](https://indico.fnal.gov/event/49686/registrations/3094/)
```
Comment (osg-cat) [inline docs/public-lectures.md]: “Click this link to register”?

**Review by osg-cat — CHANGES_REQUESTED** [review]
Good, just a few things to make the transition from email to web.

**Pull request merged**